docs/source/compute_config/oracle_functions.md

```diff
@@ -0,0 +1,130 @@
+# Oracle Functions
+
+Lithops with *Oracle Functions* as serverless compute backend.
+
+## Installation
+
+1. Install Oracle Cloud backend dependencies:
+```
+python3 -m pip install lithops[oracle]
+```
+
+2. Access to your [Oracle Cloud Console](https://cloud.oracle.com/) and activate your Functions service instance.
+
+## Configuration
+
+
+### Creating a Dynamic Group
+
+1. **Sign in to the Oracle Cloud Console.** 
+
+2. **Open the navigation menu.** Under Identity & Security , go to Polices and then click **Domains**.
+
+3. Click on the current Domain (It's called **Default** by default) and then click on **Dynamic groups**
+
+3. **Click Create Dynamic Group.**
+
+4. **In the Create Dynamic Group dialog box:**
+
+    - Give your dynamic group a **Name** and **Description**.
+  
+    - In the **Matching Rule** box, paste your rule:
+
+        ```
+        ALL {resource.type = 'fnfunc', resource.compartment.id = 'ocid1.tenancy.oc1..aaaaaaaaedomxxeig7qoo5fmbbvkdlbmp6dsdl74sh2so32zk3wxnc2dosla'}
+        ```
+    
+5. **Click Create** to create the dynamic group.
+
+### Creating a Policy for the Dynamic Group
+
+Now that the dynamic group is set up, you'll need to create a policy that allows this group to manage resources.
+
+1. **Open the navigation menu again.** Under Governance and Administration, go to Identity, and then click **Policies**.
+
+2. **Choose your compartment.**
+
+3. **Click on the Create Policy button.**
+
+4. **In the Create Policy dialog box:**
+
+    - Give your policy a **Name** and **Description**.
+    
+    - In the **Statement** box, input the policy statement that grants permissions. For example:
+
+        ```
+        Allow dynamic-group function_compartment to manage all-resources in tenancy
+        ```
+    
+    Remember to replace `function_compartment` with the name of the dynamic group you just created.
+    
+5. **Click Create** to create the policy.
+
+Now, your Oracle Functions have the necessary permissions to manage resources in your Oracle Cloud Infrastructure tenancy.
+
+
+
+1. Navigate to the Oracle Cloud Console. If you haven't already done so, follow the instructions in the Oracle documentation to generate and download the necessary API signing keys.
+
+2. Access your Oracle Functions dashboard, and choose your preferred region.
+
+3. Create a new subnet in the Virtual Cloud Network (VCN) section. If you haven't set up a subnet yet, follow the instructions in the Oracle documentation to create one. Subnet creation is mandatory.
+
+4. Edit your Lithops config and add the following keys:
+
+```yaml
+lithops:
+    backend: oracle_f
+oracle:
+    storage_bucket : <STORAGE_BUCKET>
+    user : <USER>
+    key_file : <KEY_FILE>
+    fingerprint : <FINGERPRINT>
+    tenancy : <TENANCY>
+    region : <REGION>
+    compartment_id: <COMPARTMENT_ID>
+    namespace_name : <NAMESPACE_NAME>
+oracle_f:
+    max_workers: <MAX_WORKERS>
+    runtime: <RUNTIME>
+    runtime_memory: <RUNTIME_MEMORY>
```

Comment:
You can remove max_workers, runtime and runtime_memory from here if they are not mandatory